### ISSUE TITLE
PD-Input readonly: true still has focus style (yellow)

### ISSUE THREAD

**Issue by CuriaPLUS** [Lambda-IT/parlamentsdienste-components#119]
If the pd-input component is readonly it still has the yellow background if it is focused:
<img width="1137" alt="Screenshot 2022-01-17 at 09 05 01" src="https://user-images.githubusercontent.com/77280472/149730904-4320fa84-9562-449d-8b6f-dc27cd621f99.png">

**Comment by thomasschneiter** [comment]
@CuriaPLUS This is by design as a default readonly input can still be focused. See: `<input type="text" readonly />`.

There are currently some components which are not focusable when readonly:
- pd-dropdown
- pd-datepicker
- pd-slider

We tried to create a readonly design for all of the input components but some of them create problems if they are still focusable so we disabled them in the background. 
For example the slider input doesn't support a readonly attribute, so you were still able to focus and move it by keyboard when enabled.

All the other input elements are focusable when readonly which is the expected functionality.
